### ISSUE TITLE
[QA] Several collaborators appear in the subfolders

### ISSUE THREAD

**Issue by davitol** [owncloud/ocis#574]
Tested with
 * ocis-1.0.0-rc1 via https://github.com/owncloud-docker/compose-playground/blob/master/examples/hetzner-deploy/make_ocis_eos_compose_test.sh
 * firefox and Chrome in Mac

### Steps to reproduce

1. user A shares a file with user B
2 . user B accepts the share and file appears
2. User B creates a folder in home path
3. Browse into the folder
4. Create a subfolder

### Expected behavior

Collaborators shown should be only the own user

### Current behavior

It appears 2 collaborators, user A and user B

![collaboratorsFolder](https://user-images.githubusercontent.com/10596560/93185632-f002c300-f73d-11ea-8b55-44c418f89e97.gif)

**Comment by PVince81** [comment]
yes, this is by design in Phoenix.
you see the file owner also as a collaborator

the purpose is so that anyone can see who else has access to the resource, including self

**Comment by PVince81** [comment]
ok, so according to the report it shows the wrong collaborators on unrelated folders, likely a bug with EOS.

**Comment by labkode** [comment]
@ishank011 didn't you find something similar?

**Comment by ishank011** [comment]
Yes, whenever a share is created with a user, all resources of the recipient show the share creator as the owner. And it's not in EOS, it's seen with owncloud file system as well https://github.com/owncloud/phoenix/issues/3895

**Comment by PVince81** [comment]
but according to the bug report, it even shows the share creator on non-shares resources, as if the OCS share query was returning wrongly filtered results

**Comment by ishank011** [comment]
Yeah exactly. For all of the recipient's resources, it shows the share creator as the owner. I'm guessing the bug is that it is supposed to show that only for the shared resource, but it displays the same for all resources. So the OCS queries are probably correct, but phoenix uses the wrong set of results.

**Comment by davitol** [comment]
> Yeah exactly. For all of the recipient's resources, it shows the share creator as the owner. I'm guessing the bug is that it is supposed to show that only for the shared resource, but it displays the same for all resources. So the OCS queries are probably correct, but phoenix uses the wrong set of results.

Yes, that is exactly the behavior I wanted to explain in the open post

**Comment by PVince81** [comment]
but it works fine when using OC 10 as a backend, so I doubt it's about filtering on the JS side.

needs further research
